packages/test/bin/tfm-test.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+
+const commander = require('commander');
+const { version } = require('../package.json');
+const path = require('path');
+
+const { runScript, remainingArgs } = require('../src/helpers');
+
+const testRoot = path.resolve(__dirname, '../');
+const program = new commander.Command();
```

Comment:
Will make sense to import `commander` as `const { Command } = require('commander');` so you can do `new Command()`